train_supervised.py

```diff
@@ -47,36 +49,35 @@
 testloader = torch.utils.data.DataLoader(testset, batch_size=test_batch_size,
                                          shuffle=False, num_workers=2)
 
-#define the device
+# define the device
 device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
 
 # define the model
-#model = wide_resnet.Wide_ResNet(depth=depth, widen_factor=widen_factor, dropout_rate=dropout_rate, num_classes=num_classes)
+# model = wide_resnet.Wide_ResNet(depth=depth, widen_factor=widen_factor, dropout_rate=dropout_rate, num_classes=num_classes)
 model = resnet_official.wrn_28_2().to(device)
 
-#define the optimizer and criterion
+# define the optimizer and criterion
 criterion = nn.CrossEntropyLoss()
-optimizer = optim.Adam(model.parameters(), lr=learning_rate, betas=[.9, .999])
-
+optimizer = optim.Adam(model.parameters(), lr=learning_rate, betas=(.9, .999))
```

Comment:
Unless you specifically need some functionality of a python list, like `append`ing or `extend`ing, always use tuples as containers in python.  Immutability is highly preferred to avoid headaches.